osu.Game/Overlays/Settings/Sections/SkinSection.cs

```diff
@@ -231,7 +231,7 @@ public partial class DeleteSkinButton : DangerousSettingsButton
             [BackgroundDependencyLoader]
             private void load()
             {
-                Text = "Delete";
+                Text = CommonStrings.Delete;
```

Comment:
Web already has `CommonStrings.ButtonsDelete`. The class name will conflict, so do this:
https://github.com/ppy/osu/blob/3f211e33b8217490c4fbc818f9319e65f338711b/osu.Game/Online/Leaderboards/LeaderboardScore.cs#L38